README.md

```diff
@@ -1,10 +1,13 @@
-[![Build Status](https://travis-ci.org/lightblue-platform/lightblue-core.svg?branch=master)](https://travis-ci.org/lightblue-platform/lightblue-core) [![Coverage Status](https://coveralls.io/repos/lightblue-platform/lightblue-core/badge.png?branch=master)](https://coveralls.io/r/lightblue-platform/lightblue-core?branch=master) [![Stories in Ready](https://badge.waffle.io/lightblue-platform/lightblue.svg?label=ready&title=Ready)](http://waffle.io/lightblue-platform/lightblue)
```

Comment:
Sorry, let me put this other way. I think that we should put a reference to waffle somewhere. I didn't find any link for that and I think it is important (maybe we could put on the overview repo) , what do you think?. Sorry about the misunderstanding